package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "name": "react-resizable-panels-repo",
```

Comment:
it was a Preconstruct requirement, we actually should lift this off - I don't see a strong reason why this would be hardly required for the monorepo root